src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/IDependenciesChanges.cs

```diff
@@ -10,6 +10,6 @@ public interface IDependenciesChanges
 
         ImmutableArray<IDependencyModel> AddedNodes { get; }
 
-        ImmutableArray<IDependencyModel> RemovedNodes { get; }
+        ImmutableArray<(string providerType, string dependencyId)> RemovedNodes { get; }
```

Comment:
Well I guess we get to have the "should we use tuples in a public api" discussion :)

I would prefer if we defined our own stuct to contain this infromation